Sources/FoundationEssentials/FileManager/FileManager+Directories.swift

```diff
@@ -207,6 +234,35 @@ extension _FileManagerImpl {
         withIntermediateDirectories createIntermediates: Bool,
         attributes: [FileAttributeKey : Any]? = nil
     ) throws {
+#if os(Windows)
+        try path.withNTPathRepresentation { pwszPath in
+            if createIntermediates {
+                var isDirectory: Bool = false
+                if fileExists(atPath: path, isDirectory: &isDirectory) {
```

Comment:
```suggestion
                if fileManager.fileExists(atPath: path, isDirectory: &isDirectory) {
```

This will allow subclasses to override the behavior of this `fileExists` call